tgui/packages/tgui/interfaces/CrossingSignal.tsx

```diff
@@ -0,0 +1,52 @@
+import { useBackend } from '../backend';
+import { Section, LabeledList, Flex } from '../components';
+import { Window } from '../layouts';
+
+type TransportData = {
+  sensorStatus: boolean;
+  operatingStatus: number;
+  inboundPlatform: number;
+  outboundPlatform: number;
+};
+
+type Props = {
+  context: any;
+};
+
+export const CrossingSignal = (props, context) => {
+  const { act, data } = useBackend<TransportData>(context);
+
+  const { sensorStatus, operatingStatus, inboundPlatform, outboundPlatform } =
+    data;
+
+  return (
+    <Window title="Crossing Signal" width={400} height={175} theme="dark">
+      <Window.Content>
+        <Flex direction="row">
```

Comment:
You should be able to remove flex and flex item- these are only used if you're fitting multiple things on screen and need to organize them.

If you're adding more, this would also just need to be replaced with Stack (flex is deprecated)

Reply:
Thanks.